app/modules/sightings/parameters.py

```diff
@@ -162,11 +162,13 @@ def remove(cls, obj, field, value, state):
                 raise HoustonException(
                     'Sighting patch remove encounters must be passed headers in state field'
                 )
-            delete_cascade_individual = state['headers'].get(
-                'x-allow-delete-cascade-individual', False
+            delete_cascade_individual = (
```

Comment:
and here pass state = {'delete-cascade-individual': bool, 'delete-cascade-sighting': value}